src/static/js/views/faq.html

```diff
@@ -24,7 +24,10 @@ <h4>Frequently Asked Questions</h4>
         <p>Currently, Software Discovery Tool contains data for the following operating systems:
         <ul>
             <li>IBM z/OS</li>
-            <li>Debain 10(buster)</li>
+            <li>Debian 10(buster)</li>
+			<li>ClefOS 7</li>
+			<li>OpenSUSE Tumbleweeb</li>
+			<li>OpenSUSE Leap 15.3</li>
```

Comment:
While you're in here, maybe fix up the indentations?

Reply:
ahh yeah that's `git diff` showing in that format.
This is the actual screenshot from the file:
![image](https://user-images.githubusercontent.com/42383989/126808744-319c2194-348c-4120-9083-a9c1585a72f4.png)

Reply:
Looks like it's a tabs versus spaces issue, and the tab spacing in your editor is smaller than in the git diff. Debian uses spaces, the others are using tabs. We should probably pick one.